### PR TITLE
Refactor device-quirks to work with the refactored frzr

### PR DESCRIPTION
**Pull request by NeroReflex**
This PR allow installation of device-quirks in a retro-compatible for archlinux users in a way that can take advantage of the new frzr refactor.

### CODE REVIEW

File: usr/share/device-quirks/scripts/VENDORS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+AOKZOE::aokzie
```
Comment (pastaq) [inline usr/share/device-quirks/scripts/VENDORS:1]: typo

Comment (CVMagic) [reply usr/share/device-quirks/scripts/VENDORS:1]: I will have to go through all of it to catch these minor items like typos once I am done converting over the last two remaining manufacture scripts, then on to testing each script for each vendor/model in a VM just to make sure they are working correctly. They should as I have been testing each separately, but you never know.

File: usr/share/device-quirks/scripts/chips/resources/MT7922/systemd-suspend-mods.conf
Comment (pastaq) [inline usr/share/device-quirks/scripts/chips/resources/MT7922/systemd-suspend-mods.conf:1]: This file isn't specific to one module, we're just only using it for one module at the moment.

**Comment by pastaq** [comment]
If this is ready please mark ready for review

**Review by pastaq — APPROVED** [review]
Lots of changes, but nothing immediately pops out as wrong. We'll need to fully test everything before we release.

**Pull request merged**